python/llm/src/ipex_llm/transformers/npu_model.py

```diff
@@ -438,6 +438,12 @@ def load_low_bit(cls, pretrained_model_name_or_path: str, *model_args, **kwargs)
                 model.kv_len = config_dict['kv_len']
                 model.vocab_size = config_dict['vocab_size']
                 model.logits_buffer = torch.empty(1, 1, model.vocab_size, dtype=torch.float32)
+                if model.can_generate():
+                    model.generation_config = GenerationConfig.from_pretrained(
```

Comment:
We use try except [here](https://github.com/intel-analytics/ipex-llm/blob/b39d0587555892467831fa8624c4d4c7a68f526f/python/llm/src/ipex_llm/transformers/npu_model.py#L622C9-L630C20), should we use similar operation here?

Reply:
Have updated :)